src/connection.rs

```diff
@@ -497,6 +499,7 @@ where
 }
 
 /// A handle to a multiplexed stream.
+#[derive(Debug)]
```

Comment:
This has nothing to do with the PR subject. Ok to keep it here anyway?

Reply:
I guess this is the only thing added by this PR! :D

Reply:
Forgot about this one ;-). Have added it to #26.